### PR TITLE
Enhancement: Added Django tests to github actions

### LINKED ISSUE #83: [Enhancement]: Add tests to github action
### My idea

1. Rename current github action for code style checking (also file name)
2. Add new step to it, that runs django's tests

### PR DESCRIPTION
**Pull request by Sanielia**

### CODE REVIEW

File: .github/workflows/code-style-checking.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Code quality checks
+name: Code style checking
```
Comment (Norbiros) [inline .github/workflows/code-style-checking.yml]: It isn't really `style checking` only ruff does that...

**Review by Norbiros — CHANGES_REQUESTED** [review]

File: .github/workflows/code-style-checking.yml (hunk 1 above)
Comment (Sanielia) [reply .github/workflows/code-style-checking.yml]: image

Comment (Norbiros) [reply .github/workflows/code-style-checking.yml]: I suggested renaming the action responsible for style checking, not renaming it to 'Code style checking'...

**Review by wojpo — DISMISSED** [review]

File: .github/workflows/code-and-dependencies-checks.yml [hunk 2]
```diff
@@ -13,18 +13,30 @@ jobs:
         uses: chartboost/ruff-action@v1
         with:
           args: check
-  poetry-checks:
+  poetry-dependencies-and-Django-code-tests:
```
Comment (goteusz-maszyk) [inline .github/workflows/code-and-dependencies-checks.yml]: Couldn't you have come up with even a longer name?

**Review by goteusz-maszyk — CHANGES_REQUESTED** [review]

File: .github/workflows/code-and-dependencies-checks.yml (hunk 2 above)
Comment (Sanielia) [reply .github/workflows/code-and-dependencies-checks.yml]: then how should I name it?

Comment (goteusz-maszyk) [reply .github/workflows/code-and-dependencies-checks.yml]: code-tests/code-checks

Comment (Sanielia) [reply .github/workflows/code-and-dependencies-checks.yml]: but id doesn't only check code, but also dependencies

Comment (wojpo) [reply .github/workflows/code-and-dependencies-checks.yml]: That name is readable and pretty good. Don't listen to this Java stan, Kubuś. You could maybe think of something shorter, but I have no ideas.

Comment (Norbiros) [reply .github/workflows/code-and-dependencies-checks.yml]: Why do you want to include `Django` and `poetry` in the name?

Comment (Sanielia) [reply .github/workflows/code-and-dependencies-checks.yml]: I thought it was better if name said what is used to do job, like with first job name where ruff is included

**Review by Norbiros — APPROVED** [review]

**Review by wojpo — APPROVED** [review]

**Review by goteusz-maszyk — APPROVED** [review]

**Pull request merged**